packages/button/button.scss

```diff
@@ -14,29 +14,29 @@ $button-text-size--compact: rem(14px);
 $button-border-width: rem(1px);
 $button-padding: $component-spacing--xl;
 
-$focus-ring-distance: rem(2px);
+$focus-ring-distance: rem(4px);
 $focus-ring-width: rem(2px);
 
 $animation-timing: 100ms cubic-bezier(0.6, 0.2, 0.35, 1);
 $hover-elevation-distance: -0.25rem;
 
-$button-border-color: $svart;
-$button-text-color: $svart;
+$button-border-color: $granitt;
+$button-text-color: $granitt;
 $button-background-color: transparent;
-$button-focus-color: $info;
-$button-primary-text-color: $hvit;
-$button-primary-background-color: $svart;
+$button-focus-color: $focus-color;
```

Comment:
Antar $focus-color er granitt fargen. Er $focus-color noe som blir konsistent brukt der det bør brukes eller bør vi droppe denne variabelen? Forslag @piofinn ?

Reply:
Tror ikke det er så konsekvent som det burde være. I utgangspunktet hadde det sikkert vært greit å ha en variabel for det (eller kanskje to, en for dark mode også) som brukes konsekvent, slik at man unngår forskjeller og kan endre den overalt i et sveip 😊

Reply:
Hmm.. Men innser jo nå at det er to forskjellige mønster for fokus/hover; ett der man går fra lys til mørk, og ett der man går fra mørk til lys. Så blir kanskje litt mange variabler å holde styr på 🤔

Reply:
endret til granitt